### PR TITLE
Fix Content-Type check

### LINKED ISSUE #435: If Accept request header contains extra values, 409 is returned
Getting ExtCells with `Accept: application/json, text/plain, */*` results in 409 error.

Request:

```
GET http://cell.example.com/__ctl/ExtCell HTTP/1.1
Accept: application/json, text/plain, */*
Authorization: Bearer ...
```

Response:

```
{
  "code": "PR415-OD-0001",
  message: {
    "lang": "en",
    "value": "Unsupported media type:[application/json, text/plain, */*]."
  }
}
```

Related code: https://github.com/personium/personium-core/blob/bd5568adad87b38a8788fe27dd77aed98c815d51/src/main/java/io/personium/core/rs/odata/AbstractODataResource.java#L205

`decideOutputFormatFromHeaderValues` should ignore unacceptable media type.

### PR DESCRIPTION
**Pull request by tochi-y**
Fix #435

### CODE REVIEW

File: src/main/java/io/personium/core/rs/odata/AbstractODataResource.java [hunk 1]
```diff
@@ -191,21 +191,16 @@ private MediaType decideOutputFormatFromQueryValue(String format) {
...
-            if (isAcceptXml(accept)) {
-                mediaType = MediaType.APPLICATION_ATOM_XML_TYPE;
-            } else if (isAcceptJson(accept)) {
-                if (mediaType == null) {
-                    mediaType = MediaType.APPLICATION_JSON_TYPE;
-                }
-            } else {
-                throw PersoniumCoreException.OData.UNSUPPORTED_MEDIA_TYPE.params(acceptHeaderValue);
-            }
+        String[] types = Stream.of(acceptHeaderValue.split(","))
```
Comment (Yonezawa-T2) [inline src/main/java/io/personium/core/rs/odata/AbstractODataResource.java]: This should be `acceptHeaderValue.split("[ \t]*,[ \t]*")`.
https://tools.ietf.org/html/rfc7231#section-5.3.2
https://tools.ietf.org/html/rfc7230#section-7
https://tools.ietf.org/html/rfc7230#section-3.2.3

File: src/main/java/io/personium/core/rs/odata/AbstractODataResource.java [hunk 2]
```diff
@@ -214,12 +209,8 @@ private MediaType decideOutputFormatFromHeaderValues(String acceptHeaderValue) {
      * @return String up to semicolon
      */
     private String truncateAfterSemicolon(String source) {
-        String result = source;
-        int index = source.indexOf(";");
-        if (index >= 0) {
-            result = source.substring(0, index);
-        }
-        return result;
+        String[] splited = source.split(";");
```
Comment (Yonezawa-T2) [inline src/main/java/io/personium/core/rs/odata/AbstractODataResource.java]: This should be `source.split("[ \t]*;")` or `source.split("[ \t]*;", 2)`.

https://tools.ietf.org/html/rfc7231#section-5.3.2
https://tools.ietf.org/html/rfc7230#section-3.2.3

**Comment by tochi-y** [comment]
@Yonezawa-T2 Thank you for your comments. I will modify them as you commented.

File: src/main/java/io/personium/core/rs/odata/AbstractODataResource.java (hunk 1 above)
Comment (tochi-y) [reply src/main/java/io/personium/core/rs/odata/AbstractODataResource.java]: I modified it. If you have time, could you review it?

File: src/main/java/io/personium/core/rs/odata/AbstractODataResource.java (hunk 2 above)
Comment (tochi-y) [reply src/main/java/io/personium/core/rs/odata/AbstractODataResource.java]: I modified it, too.

File: src/test/java/io/personium/test/unit/core/rs/odata/DecideOutputFormatTest.java [hunk 3]
```diff
@@ -112,6 +136,30 @@
...
+
+    /**
+     * acceptのタブとセミコロン以降を無視すること.
+     */
+    @Test
+    public final void acceptのタブとセミコロン以降を無視すること() {
+        ODataEntityResource odataEntityResource = new ODataEntityResource();
+
+        MediaType type = odataEntityResource.decideOutputFormat(
+                "application/xml\t;q=0.9,*/\t;q=0.8", null);
```
Comment (Yonezawa-T2) [inline src/test/java/io/personium/test/unit/core/rs/odata/DecideOutputFormatTest.java]: Just a nitpicking: `*/\t` → `*/*\t`

File: src/main/java/io/personium/core/rs/odata/AbstractODataResource.java (hunk 1 above)
Comment (Yonezawa-T2) [reply src/main/java/io/personium/core/rs/odata/AbstractODataResource.java]: I found no problems.

File: src/test/java/io/personium/test/unit/core/rs/odata/DecideOutputFormatTest.java (hunk 3 above)
Comment (tochi-y) [reply src/test/java/io/personium/test/unit/core/rs/odata/DecideOutputFormatTest.java]: Thank you for you review. I modified it. After automatic testing, I will merge this PR.

**Pull request merged**